VideoLocker/src/main/java/org/edx/mobile/task/CreateResponseTask.java

```diff
@@ -1,25 +0,0 @@
-package org.edx.mobile.task;
-
-import android.content.Context;
-import android.support.annotation.NonNull;
-
-import org.edx.mobile.discussion.DiscussionComment;
-import org.edx.mobile.discussion.ResponseBody;
-
-public abstract class CreateResponseTask extends
-Task<DiscussionComment> {
```

Comment:
If the `createResponse()` method is not being used, then is should be removed from the `DiscussionService` definition, along with the `ResponseBody` parameter utilized there.